run-in-retro-term.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+#
+# cool-retro-term support for tetris.sh
+# Author: Rojen Zaman <rojen@riseup.net>
+# License: WTFPL
+
+set -u                                                    # return error if string null
+stdout="/dev/null"                                        # default stdout, optional verbose mode is /dev/stdout
+script_dir=$(cd $(dirname $0) && pwd)                     # script dir, it is require for cool-retro-term
```

Comment:
Let's change this to the
```
script_dir=$(dirname $(readlink -f $0))
```
The reason is that if somebody will put symlink to the script in $HOME/bin current code will set $script_dir to $HOME/bin instead of the correct directory. With readlink -f this will work as expected.

Reply:
Oh, and please change this in the tetris.sh as well.